Mathlib/GroupTheory/Solvable.lean

```diff
@@ -54,6 +54,19 @@ theorem derivedSeries_normal (n : ℕ) : (derivedSeries G n).Normal := by
 theorem derivedSeries_one : derivedSeries G 1 = commutator G :=
   rfl
 
+theorem derivedSeries_antitone : Antitone (derivedSeries G) := by
+  refine antitone_nat_of_succ_le fun n => ?_
+  simp only [derivedSeries_succ]
+  rw [← Subgroup.map_subtype_commutator]
+  exact Subgroup.map_subtype_le _
```

Comment:
Can you add `commutator_le_self` to `Abelianization.lean` (immediately after `map_subtype_commutator`)?
```
variable {G} in
theorem Subgroup.commutator_le_self (H : Subgroup G) : ⁅H, H⁆ ≤ H := by
  refine H.map_subtype_commutator.symm.trans_le (map_subtype_le (_root_.commutator H))
```
```suggestion
theorem derivedSeries_antitone : Antitone (derivedSeries G) :=
  antitone_nat_of_succ_le fun n => (derivedSeries G n).commutator_le_self
```